src/bundle/Resources/views/admin/content_type/list.html.twig

```diff
@@ -72,13 +72,13 @@
                         <use xlink:href="{{ ez_content_type_icon(content_type.identifier) }}"></use>
                     </svg>
                 </td>
-                <td class="ez-table__cell ez-table__cell--after-icon">
+                <td class="ez-table__cell ez-table__cell--name ez-table__cell--after-icon">
                 {% set view_url = path('ezplatform.content_type.view', {
                         'contentTypeGroupId': content_type_group.id,
                         'contentTypeId': content_type.id
                     }) %}
 
-                    <a href="{{ view_url }}">{{ content_type.name }}</a>
+                    <a href="{{ view_url }}" title="{{ content_type.name }}">{{ content_type.name }}</a>
```

Comment:
Sometimes the tag "a" is one line and sometimes multi, maybe we'll set a standard for it?

Reply:
It goes multi-line when the line is too long ;)